programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1137,6 +1151,35 @@ static int run_tunnel(uv_loop_t *ziti_loop, uint32_t tun_ip, uint32_t dns_ip, co
     tun = tun_open(ziti_loop, tun_ip, dns_ip, ip_range, tun_error, sizeof(tun_error));
 #elif _WIN32
     tun = tun_open(ziti_loop, tun_ip, dns_ip, ip_range, tun_error, sizeof(tun_error));
+
+    bool nrpt_effective = is_nrpt_policies_effective(get_dns_ip());
+    if (!nrpt_effective || get_add_dns_flag()) {
+        ZITI_LOG(INFO, "Enable DNS for %s", get_dns_ip());
+        set_dns(tun->handle, dns_ip);
+    }
+    if (nrpt_effective) {
+        model_map *domains = get_connection_specific_domains();
+        const char *key;
+        bool status;
+        model_map *normalized_domains = calloc(1, sizeof(model_map));
+        model_map_iter it = model_map_iterator(domains);
+        while (it != NULL) {
+            char *key = model_map_it_key(it);
+            model_map_set(normalized_domains, normalize_host(key), true);
```

Comment:
`true` is not a valid value for a pointer argument. Is the value of this map ever used? I think it isn't - it looks like the map is being used as a list of keys, so you could set the values to NULL.